src/ado2gh/Commands/DownloadLogsCommand.cs

```diff
@@ -127,21 +136,36 @@ HttpDownloadService httpDownloadService
                 _log.LogWarning($"Overwriting {migrationLogFile} due to --overwrite option.");
             }
 
-            _log.LogInformation($"Downloading log for repository {githubRepo} to {migrationLogFile}...");
-
             var githubApi = _githubApiFactory.Create(githubApiUrl, githubPat);
-            var logUrl = await githubApi.GetMigrationLogUrl(githubOrg, githubRepo);
+            string logUrl;
 
-            if (logUrl == null)
+            while (true)
```

Comment:
Instead of a `while (true)` it's be better to do `while (logUrl is null)` (assuming we shouldn't expect to get an empty or whitespace string). And of course you can add a timeout to the predicate too.

Reply:
I replaced the `while(true)` with a for loop in 20b4a78bf1a10bbec8e1cb1e1bbf2ff801b7d7c4.  How's this look?